website/project/views/node.py

```diff
@@ -449,6 +450,11 @@ def project_statistics(auth, node, **kwargs):
     return _view_project(node, auth, primary=True)
 
 
+@must_be_valid_project
+@must_be_contributor_or_public
+def project_statistics_redirect(auth, node, **kwargs):
+    return redirect(node.web_url_for("project_statistics"))
```

Comment:
Sorry, but I take back my previous recommendation to use `node.web_url_for`. We want `/abc12/statistics/` to redirect to `/abc12/analytics`, NOT `/project/abc12/analytics`.

I will make this change when I merge.